spring-cloud-netflix-core/src/test/java/org/springframework/cloud/netflix/metrics/servo/ServoMetricReaderTests.java

```diff
@@ -36,7 +36,7 @@ public void singleCompositeMonitorYieldsMultipleActuatorMetrics() {
 				metricNames.get(0));
 		assertEquals("metricName(statistic=max,type=GAUGE,unit=MILLISECONDS)",
 				metricNames.get(1));
-		assertEquals("metricName(statistic=min,type=GAUGE,unit=MILLISECONDS)",
+		assertEquals("metricName(statistic=totalOfSquares,type=NORMALIZED,unit=MILLISECONDS)",
```

Comment:
@spencergibb This is expected.  Here is the reasoning that Daniel Muino provided when this change was made:

The BasicTimer in Servo is used to measure short-duration events.  To accomplish this, it publishes four different values:

`totalTime` (sum of all durations)
`count` (number of times an event happened)
`max` (maximum duration of an event)
`min` (minimum duration of an event, or 0 if no events occurred)

Three of these values have proven useful over time: the `totalTime` and `count` can be used to get an accurate average across arbitrary groups of metrics, and the `max` statistic is used for detecting outlier behavior in clusters.  The `min` value does not have a good use case: it is not used in any alerts, and it does not aggregate well (if any nodes report zero, then the min is zero).

The next release of Servo will replace the `min` statistic with a different one: `totalOfSquares`.  We hope this new statistic will prove more useful to a majority of our users.  With `totalOfSquares`, we can compute an accurate standard deviation across arbitrary groups of metrics.